### ISSUE TITLE
VoiceOver Rotor Operations Lack Feedback

### ISSUE THREAD

**Issue by luomo-pro** [flutter/flutter#155063]
### Steps to reproduce

As I mentioned in #151029.

When using VoiceOver's rotor operations to perform actions such as paste, delete, etc., there is a lack of feedback after the operation is completed.

For example, with the native text fields, after pasting, it should read out the pasted text.
After selecting all, it should read out "XX has been selected."

However, in Flutter, there is no feedback information at all.

Please use the latest master version for testing.

### Expected results

There should be corresponding feedback information after each operation.

### Actual results

There is no feedback information at all.

### Code sample

<details open><summary>Code sample</summary>

```dart
import 'package:flutter/material.dart';

void main() {
  runApp(const MyApp());
}

class MyApp extends StatelessWidget {
  const MyApp({Key? key}) : super(key: key);

  @override
  Widget build(BuildContext context) {
    return MaterialApp(
      home: Scaffold(
        appBar: AppBar(title: const Text('VoiceOver Test')),
        body: const Center(
          child: Padding(
            padding: EdgeInsets.all(16.0),
            child: TextField(
              decoration: InputDecoration(
                hintText: 'Enter text here',
              ),
            ),
          ),
        ),
      ),
    );
  }
}
```

</details>

### Flutter Doctor output

<details open><summary>Doctor output</summary>

```console
Flutter (Channel master, 3.26.0-1.0.pre.97
```

</details>

**Comment by luomo-pro** [comment]
cc @hellohuanlin

**Comment by darshankawar** [comment]
@luomo-pro 

> VoiceOver's rotor operations to perform actions such as paste, delete, etc.,

I tried to follow this by using the rotor but I couldn't find an option or know how to paste, delete using it. Can you provide details about it ?

I also tried following [these](https://github.com/flutter/flutter/issues/151029#issuecomment-2311995421) steps you mentioned but unable to carry out the rotor actions to paste.

**Comment by luomo-pro** [comment]
@darshankawar
First, ensure you are using the latest master version.
Then, follow these steps:

1. Use VoiceOver to touch the text field.
2. Double-tap to activate it, which will bring up the keyboard.
3. Use the VoiceOver rotor gesture.
To explain simply, the rotor gesture is performed by placing two fingers on the screen and rotating them as if turning a dial. As you rotate, you'll hear various options such as "Characters", "Words", "Edit", and many others.
4. When you hear "Edit", use one finger to swipe down. You'll hear options like "Paste", "Select All", etc. Double-tap to execute the chosen action.
Note that "Paste" will only appear if you've copied some text, and "Select All" will only appear when there's text in the text field.

**Comment by darshankawar** [comment]
Thanks for the update. Upon following above steps and running them in iphone 6s device, I was able to replicate the reported behavior. 

`Master: 3.26.0-1.0.pre.72`

**Comment by darshankawar** [comment]
Upon trying it again for few more times, I was able to paste the selected text and it does give a feedback of it, as verified in https://github.com/flutter/flutter/issues/155061#issuecomment-2348130116

@luomo-pro please check above.

**Comment by luomo-pro** [comment]
@darshankawar
Receiving feedback like "pasted from notes" after a paste operation is incorrect.
In native iOS, after pasting, it tells you what content was pasted, not where it was pasted from.
Additionally, besides pasting, there's also the select all operation.
In native, after selecting all, it also feedback "XX selected".

**Comment by chunhtai** [comment]
my guesses is the FlutterTextInputPlugin overrides UIResponderStandardEditActions and maybe voiceover expected synchronize state changes. I wonder if we really need to override these method? At least for copy, it TextView should handles the copy itself if we sets correct `selectedTextRange` I assume? cc @hellohuanlin who worked on ios flutter text plugin 

cc @hua

**Comment by hellohuanlin** [comment]
> I wonder if we really need to override these method? 

@chunhtai I don't think we can skip the implementation of `copy` tho - we have to update text input's internal states. Also if we skip, i believe the default behavior is a no-op implementation
